src/main/resources/com/mesosphere/velocity/marathon/MarathonRecorder/help-forceUpdate.html

```diff
@@ -0,0 +1,16 @@
+<div>
+    <p>
+        This parameter decides how to handle updating a Marathon application that has an active deployment.
+    </p>
+    <p>
+        If this is set to <code>true</code> (checked), then a new deployment will be forced and the current deployment
+        canceled. If this is set to <code>false</code> (unchecked), then this plugin will attempt to update the Marathon
+        application every couple of seconds until a timeout is reached. If the active deployment does not complete within
+        the time frame, then an error will be returned and the job marked as failed.
```

Comment:
Got it. In which case it sounds like we should re-word this (because we don't want to conflate the failure of this plugin to create a deployment with the failure of the deployment itself).